test/docker-e2e/e2e_test.go

```diff
@@ -41,6 +44,61 @@ func (s *CelestiaTestSuite) SetupSuite() {
 	s.client, s.network = tastoradockertypes.DockerSetup(s.T())
 }
 
+// ExecuteNodeCommand executes a command on a chain node with common parameters automatically added.
+// This reduces boilerplate in tests by setting common flags like --home, --node, --fees, etc.
+func (s *CelestiaTestSuite) ExecuteNodeCommand(ctx context.Context, chainNode tastoratypes.ChainNode, cmd ...string) (string, string, error) {
+	s.Require().Greater(len(cmd), 0, "command must not be empty")
+
+	var finalCmd []string
+
+	if cmd[0] == "celestia-appd" {
+		s.Require().Greater(len(cmd), 1, "celestia-appd must have at least one subcommand")
+		cmd = cmd[1:]
+	}
+
+	isTxCommand := cmd[0] == "tx"
+	isKeysCommand := cmd[0] == "keys"
+
+	// Common flags for all commands
+	if !slices.Contains(cmd, "--home") {
+		finalCmd = append(finalCmd, "--home", homeDir)
+	}
+
+	// Add --node flag for all commands except key management commands
+	if !isKeysCommand && !slices.Contains(cmd, "--node") {
+		hostname, err := chainNode.GetInternalHostName(ctx)
+		if err != nil {
+			return "", "", err
+		}
+		finalCmd = append(finalCmd, "--node", fmt.Sprintf("tcp://%s:26657", hostname))
+	}
+
+	// Add --chain-id flag for all commands except key management commands
+	if !isKeysCommand && !slices.Contains(cmd, "--chain-id") {
+		finalCmd = append(finalCmd, "--chain-id", appconsts.TestChainID)
+	}
+
+	// Transaction-specific flags
+	if isTxCommand {
+		if !slices.Contains(cmd, "--fees") {
+			finalCmd = append(finalCmd, "--fees", "200000utia")
+		}
+
+		if !slices.Contains(cmd, "--keyring-backend") {
+			finalCmd = append(finalCmd, "--keyring-backend", "test")
+		}
+
+		if !slices.Contains(cmd, "--yes") {
+			finalCmd = append(finalCmd, "--yes")
+		}
+	}
+
+	finalCmd = append([]string{"celestia-appd"}, append(cmd, finalCmd...)...)
```

Comment:
[not blocking comment]

the naming of this command then prepending things to it is wonky

just name it cli command or something